build/Targets/Settings.props

```diff
@@ -31,6 +31,12 @@
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)' == ''">$(RepoRoot)Binaries\Obj\</BaseIntermediateOutputPath>
     <IntermediateOutputPath>$(BaseIntermediateOutputPath)$(Configuration)\$(MSBuildProjectName)\</IntermediateOutputPath>
 
+    <NetStandard20LibraryVersion>2.0.0-beta-25123-01</NetStandard20LibraryVersion>
+    <NetStandard20RefPath>$(NuGetPackageRoot)/NETStandard.Library/$(NetStandard20LibraryVersion)/build/netstandard2.0/ref/</NetStandard20RefPath>
+
+    <MicrosoftCSharpVersion>4.3.0</MicrosoftCSharpVersion>
+    <MicrosoftCSharpRefPath>$(NuGetPackageRoot)/Microsoft.CSharp/$(MicrosoftCSharpVersion)/ref/netstandard1.0/</MicrosoftCSharpRefPath>
```

Comment:
Why are we specifying thes properties here?  Shouldn't they be generated into Dependencies.props?